CMakeLists.txt

```diff
@@ -20,10 +20,17 @@ endif(NOT CMAKE_BUILD_TYPE)
 option(SPM_ENABLE_SHARED "" OFF)
 option(SPM_ENABLE_TCMALLOC "" OFF)
 add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/third_party/sentencepiece EXCLUDE_FROM_ALL)
+
+if(MSVC)
+  set(FLAGS "/w")
```

Comment:
Could you use a more specific variable name such as `SENTENCEPIECE_CXX_FLAGS`?

Reply:
Sure, thought about it afterwards too :)